src/Apps/Artist/Routes/Consign/Utils/staticCSVToJSONData.ts

```diff
@@ -0,0 +1,2372 @@
+/**
+ * CSV data exported from https://docs.google.com/spreadsheets/d/1EicHOCyCozsPBkVOkQGTTUtEds7qQuacUAgLy-C2-Nk/edit#gid=0
+ *
+ * To update data:
+ *   1) Open spreadsheet link above
+ *   2) File > Export > CSV
+ *   3) Copy CSV data and paste into https://www.csvjson.com/csv2json
+ *   4) Copy results and overwrite below
+ */
```

Comment:
@pepopowitz - I added some update docs here. Anyone needing to modify or update this static data will encounter this file in the execution path. Does this satisfy your request?

Reply:
Looks great!